java/java_test.go

```diff
@@ -376,6 +376,44 @@ func testJavaWithBuilder(builder string) func(*testing.T, spec.G, spec.S) {
 				})
 			})
 
+			context("app uses opentelemetry", func() {
+				it("builds successfully", func() {
+					var err error
+					source, err = occam.Source(filepath.Join("../java", "opentelemetry"))
+					Expect(err).NotTo(HaveOccurred())
+
+					var logs fmt.Stringer
+					image, logs, err = pack.Build.
+						WithPullPolicy("never").
+						WithEnv(map[string]string{
+							"BP_OPENTELEMETRY_ENABLED": "true",
+						}).
+						WithBuilder(builder).
+						WithBuildpacks(
+							"paketo-buildpacks/java",
+							"gcr.io/paketo-buildpacks/opentelemetry",
+						).
+						WithVolumes(fmt.Sprintf("%s/.gradle:/home/cnb/.gradle:rw", home)).
+						WithGID("121").
+						Execute(name, source)
+					Expect(err).ToNot(HaveOccurred(), logs.String)
+
+					Expect(logs).To(ContainLines(ContainSubstring("Paketo Buildpack for CA Certificates")))
+					Expect(logs).To(ContainLines(ContainSubstring("Paketo Buildpack for BellSoft Liberica")))
+					Expect(logs).To(ContainLines(ContainSubstring("Paketo Buildpack for Grade")))
+					Expect(logs).To(ContainLines(ContainSubstring("Paketo Buildpack for Executable JAR")))
+					Expect(logs).To(ContainLines(ContainSubstring("Paketo Buildpack for Spring Boot")))
+					Expect(logs).To(ContainLines(ContainSubstring("Paketo OpenTelemetry Buildpack")))
+
```

Comment:
This name has been changed in https://github.com/paketo-buildpacks/opentelemetry/pull/42. Once a new release of the OpenTelemetry duildpack is cut, this test needs to be updated to check for the new name.

Reply:
This has been cut.

Reply:
Great, thanks! I'll update the test. Is there anything I can do to help fixing the test setup so that we can merge this PR?